exporter/elasticsearchexporter/model.go

```diff
@@ -146,76 +144,13 @@ func (m *encodeModel) encodeLogOTelMode(resource pcommon.Resource, resourceSchem
 
 	document.AddTraceID("trace_id", record.TraceID())
 	document.AddSpanID("span_id", record.SpanID())
-	document.AddInt("trace_flags", int64(record.Flags()))
```

Comment:
This is a breaking change, but it only affects the behavior of the `otel` mapping mode, which is marked as experimental, so the usual rules for breaking changes in beta components don't apply here.

Still, I think it's worth mentioning this in the changelog. @carsonip Can you please add a separate changelog entry with `change_type: breaking` saying that the trace flags are no longer exported in logs OTel mode?

Reply:
Updated